executor/src/witgen/machines/fixed_lookup_machine.rs

```diff
@@ -47,6 +46,27 @@ impl<T> IndexValue<T> {
     }
 }
 
+#[derive(Debug, Clone)]
+enum FixedColOrConstant<T, C> {
+    FixedCol(C),
+    Constant(T),
+}
+
+impl<T: Clone> TryFrom<&AlgebraicExpression<T>> for FixedColOrConstant<T, PolyID> {
```

Comment:
```suggestion
impl<T: Copy> TryFrom<&AlgebraicExpression<T>> for FixedColOrConstant<T, PolyID> {
```
?

Reply:
Is there a downside to using clone? I thought clone is a less strict requirement, so why not.

Reply:
it's fine, but usually, clone can be very expensive and if we need to support arbitrary clone types, we would rather store a reference (which supports copy)